roles/common/tasks/k8s.yml

```diff
@@ -191,45 +208,52 @@
   become: yes
   become_user: buildslave
   shell: |
-    az login --service-principal -u {{ azure_secrets.azure_user }} -p {{ azure_secrets.azure_key }} --tenant {{ azure_secrets.azure_tenant }}
+    echo "If this fails, check note in k8s.yml about how to setup MSI for Azure VMs"
+    az login --identity
+
+
 
 - name: Get Azure kubectl credentials
   tags:
     - azure
     - k8s
   become: yes
   become_user: buildslave
-  shell: az aks get-credentials --resource-group {{ azure_secrets.azure_resource }} --name {{ item.name }}
+  shell: az aks get-credentials --resource-group {{ item.resource_group }} --name {{ item.name }}
   loop:
-    - {name: "aks-kci-france-central"}
-    - {name: "aks-kci-us-east2"}
-    - {name: "aks-kci-uk-south"}
+    - {name: "aks-kbuild-medium-1", resource_group: "rg-kernelci-aks-us-east2"}
+    - {name: "aks-kbuild-medium-2", resource_group: "rg-kernelci-aks-s-cent-us"}
+    - {name: "aks-kbuild-medium-3", resource_group: "rg-kernelci-aks-us-west3"}
+    - {name: "aks-kbuild-big-1", resource_group: "rg-kernelci-aks-uk-south"}
 
 #
 # K8S common
 #
 # NOTE: this only needs to be done on a single host since it
 #       configures the k8s cluster, not the ansible host
 #
-- name: Get KCI auth tokens
-  tags:
-    - k8s-secrets
-    - never
-  become: yes
-  become_user: buildslave
-  include_vars:
-    file: group_vars/kci_tokens.yml
-    name: kci_tokens
-
+# 20/Jun/2023 - This is needed only on initial K8S cluster setup! But important to set secrets
+# TODO: Not sure why it is in builders ansible recipe, k8s creation should be standalone recipe
+#
+#- name: Get KCI auth tokens
+#  tags:
+#    - k8s-secrets
+#    - never
```

Comment:
I guess the `never` tag means it's never run unless you ask for it explicitly on the command line?

Reply:
I didnt wanted to take a risk breaking k8s cluster, might take a look later, also this section likely need updates too

Reply:
How about this then?  Do you have the time to check that `never` really means it's never run unless explicitly mentioned on the command line or do you want to leave it commented-out?

Reply:
Ok, i will try to check them today to make sure it doesnt run

Reply:
Have you confirmed that the `never` tag does what we want?